ship_dataset.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Aug 20 15:05:17 2019
+
+@author: j-bd
+"""
+import numpy as np
+
+from mrcnn import utils
+
+
+class ShipDataset(utils.Dataset):
+    '''class to manage the data for the training process with mrcnn'''
+    def load_ship(self, dataset_dir, subset):
+        ''' Reads pandas dataframe, extracts the annotations, and iteratively
+        calls the internal 'add_class' and 'add_image' functions to build the
+        dataset.'''
+        self.add_class("ship", 1, "ship")
+
+        for image in subset.ImageId.unique():
+            img_masks = subset.loc[subset['ImageId'] == image,
+                                   'EncodedPixels'].tolist()
+            self.add_image("ship",
+                           image_id=image,
+                           path=dataset_dir+image,
+                           width=768,
```

Comment:
It might be a good idea to create a `constants.py` file that stores all those magic numbers